src/lambda_codebase/account_processing/configure_account_alias.py

```diff
@@ -76,6 +80,7 @@ def lambda_handler(event, _):
             "adf_account_alias_config",
         )
         ensure_account_has_alias(event, role.client("iam"))
+        EVENTS.put_event(detail=json.dumps({"account_id": account_id, "alias_value": event.get("alias")}), detailType="ACCOUNT_ALIAS_CONFIGURED", resources=[account_id])
```

Comment:
Please reformat this.  https://black.readthedocs.io/en/stable/getting_started.html can automatically do this for you